composer.json

```diff
@@ -10,77 +10,97 @@
+        "surfnet/stepup-saml-bundle": "dev-feature/kill-off-travis as 6.0.8",
+        "symfony/console": "^6.3",
+        "symfony/expression-language": "^6.3",
+        "symfony/flex": "^v2.4.1.",
+        "symfony/form": "^6.3",
+        "symfony/framework-bundle": "^6.3",
+        "symfony/http-foundation": "^6.3",
         "symfony/monolog-bundle": "^3.1.0",
-        "symfony/security-bundle": "4.4.*",
-        "symfony/stopwatch": "4.4.*",
-        "symfony/translation": "4.4.*",
-        "symfony/twig-bundle": "4.4.*",
-        "symfony/web-profiler-bundle": "4.4.*",
+        "symfony/runtime": "^6.3",
+        "symfony/security-bundle": "^6.3",
+        "symfony/stopwatch": "^6.3",
+        "symfony/translation": "^6.3",
+        "symfony/twig-bundle": "^6.3",
+        "symfony/web-profiler-bundle": "^6.3",
         "symfony/webpack-encore-bundle": "^1.8",
-        "twig/extensions": "^1.5",
-        "twig/extra-bundle": "^2.12|^3.0",
-        "twig/twig": "^2"
+        "twig/extra-bundle": "^3.0",
+        "twig/intl-extra": "*",
+        "twig/twig": "^3"
     },
     "require-dev": {
-        "roave/security-advisories": "dev-latest",
-        "jakub-onderka/php-parallel-lint": "^1.0",
-        "malukenho/docheader": "^0.1.8",
+        "malukenho/docheader": "^1.0.0",
         "matthiasnoback/symfony-config-test": "^4",
         "mockery/mockery": "~1.0",
+        "overtrue/phplint": "*",
         "phpmd/phpmd": "^2.6",
-        "phpunit/phpunit": "^8",
-        "sebastian/exporter": "^3",
-        "sebastian/phpcpd": "^4",
-        "squizlabs/php_codesniffer": "^3.4",
-        "symfony/browser-kit": "4.4.*",
-        "symfony/css-selector": "4.4.*",
-        "symfony/phpunit-bridge": "^3.0"
+        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan-deprecation-rules": "*",
+        "phpunit/phpunit": "^9.5",
+        "rector/rector": "^0.18.5",
+        "roave/security-advisories": "dev-latest",
+        "slevomat/coding-standard": "^8.13",
+        "sebastian/exporter": "^4.0.5",
+        "sebastian/phpcpd": "^6.0",
+        "squizlabs/php_codesniffer": "^3.7",
+        "symfony/browser-kit": "^6.3",
+        "symfony/css-selector": "^6.3",
+        "symfony/phpunit-bridge": "^6.3"
     },
     "scripts": {
-        "test": [
-            "@lint",
+        "check": [
+            "@composer-validate",
+            "@phplint",
             "@phpmd",
             "@phpcs",
             "@phpcpd",
             "@license-headers",
-            "@phpunit",
+            "@test",
             "@security-tests",
             "@yarn-audit"
         ],
-        "lint": [
-            "@lint-php",
-            "@lint-composer"
-        ],
-        "lint-php": "vendor/bin/parallel-lint src",
-        "lint-composer": "composer validate",
+        "composer-validate": "./ci/qa/validate",
+        "phplint": "./ci/qa/phplint",
         "license-headers": "vendor/bin/docheader check src/",
```

Comment:
Can you move this to a docheader script in the qa/ci folder? See example in the GSSP projects